FHEM/00_SIGNALduino.pm

```diff
@@ -70,6 +71,7 @@ my %gets = (    # Name, Data to send to the SIGNALduino, Regexp for the answer
 #  "FAParms"  => ["fp", '.*' ],
 #  "TCParms"  => ["dp", '.*' ],
 #  "HXParms"  => ["hp", '.*' ]
+  "queryFirmware" => ["none",'none'],
```

Comment:
in availableFirmare umbenennen

Reply:
Ich werde mein bestes tun. Da das hier ebenso "neuland" ist mit manuellem korrigieren.
Prinzipiell weiß ich worum es geht und darüber las ich schon heute.

Reply:
den branch kann man auch ganz normal clonen... also muss man hier nicht im web editieren.